test/Interop/C/swiftify-import/Inputs/counted-by-noescape.h

```diff
@@ -24,3 +24,13 @@ int * __counted_by(len) __noescape returnPointer(int len);
 int * __counted_by(len1) returnLifetimeBound(int len1, int len2, int * __counted_by(len2) p __lifetimebound);
 
 void anonymous(int len, int * __counted_by(len) _Nullable __noescape);
+
+void keyword(int len, int * __counted_by(len) _Nullable func __noescape);
```

Comment:
I think we should try to test (at least some more of) these keywords: `extension`, `actor`, `init`, `open`, `var`, `is`, `as`, `in`, `guard`, and `where`; I can imagine scenarios where these are used as identifiers in C/C++.